textinput/textinput.go

```diff
@@ -473,68 +513,49 @@ func (m Model) Update(msg tea.Msg) (Model, tea.Cmd) {
 
 	switch msg := msg.(type) {
 	case tea.KeyMsg:
-		switch msg.Type {
-		case tea.KeyBackspace, tea.KeyCtrlH: // delete character before cursor
+		switch {
+		case key.Matches(msg, m.KeyMap.DeleteWordBackward):
 			m.Err = nil
-
-			if msg.Alt {
-				m.deleteWordLeft()
-			} else {
-				if len(m.value) > 0 {
-					m.value = append(m.value[:max(0, m.pos-1)], m.value[m.pos:]...)
-					if m.pos > 0 {
-						m.SetCursor(m.pos - 1)
-					}
+			m.deleteWordBackward()
+		case key.Matches(msg, m.KeyMap.DeleteCharacterBackward):
+			m.Err = nil
+			if len(m.value) > 0 {
+				m.value = append(m.value[:max(0, m.pos-1)], m.value[m.pos:]...)
+				if m.pos > 0 {
+					m.SetCursor(m.pos - 1)
 				}
 			}
-		case tea.KeyLeft, tea.KeyCtrlB:
-			if msg.Alt { // alt+left arrow, back one word
-				m.wordLeft()
-				break
-			}
-			if m.pos > 0 { // left arrow, ^F, back one character
+		case key.Matches(msg, m.KeyMap.WordBackward):
+			m.wordLeft()
```

Comment:
I think we should rename this method to match the others.

```suggestion
			m.wordBackward()
```

Reply:
done